SPECS/numpy/numpy.spec

```diff
@@ -1,131 +1,856 @@
-Summary:        Array processing for numbers, strings, records, and objects
+#uncomment next line for a release candidate or a beta
+#%%global relc rc1
+
+%global blaslib openblas
+%global blasvar p
+
+Summary:        A fast multidimensional array facility for Python
 Name:           numpy
-Version:        1.22.3
+Version:        1.23.4
 Release:        2%{?dist}
-# The custom license is inside numpy/core/src/multiarray/dragon4.c.
-License:        BSD AND ZLIB custom
+# Everything is BSD except for class SafeEval in numpy/lib/utils.py which is Python
+License:        BSD AND Python AND ASL 2.0
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
-Group:          Development/Languages/Python
-URL:            https://numpy.org/
-Source0:        https://github.com/numpy/numpy/releases/download/v%{version}/%{name}-%{version}.tar.gz
+URL:            http://www.numpy.org/
+Source0:        https://github.com/%{name}/%{name}/releases/download/v%{version}/%{name}-%{version}.tar.gz
+Source1:        https://numpy.org/doc/1.23/numpy-html.zip
```

Comment:
I'd recommend to add 1.23 in zip file name so blob store and have several versions of this guy (this will solve issue that can happen if we move to version 1.24.x later, e.g.: 1.24.x for Mariner 3.0 and still on 1.23.x for Mariner 2.0)
You might also want to automate downloading of that zip file in generate_source_tarball.sh

Reply:
e.g.: "Source1:  https://numpy.org/doc/1.23/numpy-html.zip#/numpy-html-1.23.zip"
fyi, RPM build only care about what's after the last '/'